.github/workflows/main.yml

```diff
@@ -409,10 +409,12 @@ jobs:
 
       - name: Import GPG key
         id: import_gpg
-        uses: kbst/ghaction-import-gpg@v3
+        uses: crazy-max/ghaction-import-gpg@v5.2.0
         with:
-          gpg-private-key: ${{ secrets.GPG_PRIVATE_KEY }}
+          gpg_private_key: ${{ secrets.GPG_PRIVATE_KEY }}
           passphrase: ${{ secrets.GPG_PASSPHRASE }}
+          git_user_signingkey: true
+          git_commit_gpgsign: true
```

Comment:
Please keep the changes to the action out of this PR. It may very well be required that I update the action before I can cut a new release. But then I will do so by updating my fork, not consuming it right from the upstream repository.

Reply:
done, i just didn't figure out how to use existing action at all (gave me errors) nor could find it's sources